integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperatorTest.java

```diff
@@ -403,6 +412,263 @@ public void testAutoAndCustomSitConfigOverrides() throws Exception {
+  }
+
+  /**
+   * This is the test case to cover bug 29684570.
+   *
+   * @throws Exception - if any error occurs
+   */
+  @Test
+  public void testJRFDomainMSPodCreated() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethod = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethod);
+
+    if (operator1 == null) {
+      operator1 = TestUtils.createOperator(JRF_OPERATOR_FILE_1);
+    }
+
+    JRFDomain domain1 = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> domain1Map = TestUtils.loadYaml(JRF_DOMAIN_ON_PV_WLST_FILE);
+      domain1Map.put("domainUID", "jrfmspod");
+      domain1Map.put("adminNodePort", 30706);
+      domain1Map.put("t3ChannelPort", 30026);
+      domain1Map.put("voyagerWebPort", 30310);
+      domain1Map.put("rcuSchemaPrefix", "jrfmspod");
+      domain1Map.put("initialManagedServerReplicas", 4);
+      domain1Map.put("exposeAdminNodePort", false);
+
+      // run RCU script to load db schema
+      DBUtils.runRCU(rcuPodName, domain1Map);
+
+      // create domain
+      logger.info("Creating Domain & verifying the domain creation");
+      domain1 = new JRFDomain(domain1Map);
+      domain1.verifyDomainCreated();
+
+    } finally {
+      if (domain1 != null && (JENKINS || testCompletedSuccessfully)) {
+        domain1.destroy();
+      }
+    }
+    logger.info("SUCCESS - " + testMethod);
+  }
+
+  /**
+   * This is the test case to cover bug 29683926
+   *
+   * @throws Exception - if any error occurs
+   */
+  @Test
+  public void testJRFDomainCreateDomainScriptsMountPath() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethod = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethod);
+
+    if (operator1 == null) {
+      operator1 = TestUtils.createOperator(JRF_OPERATOR_FILE_1);
+    }
+
+    JRFDomain domain1 = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> domain1Map = TestUtils.loadYaml(JRF_DOMAIN_ON_PV_WLST_FILE);
+      domain1Map.put("domainUID", "jrfcdsmp");
+      domain1Map.put("adminNodePort", 30707);
+      domain1Map.put("t3ChannelPort", 30027);
+      domain1Map.put("voyagerWebPort", 30311);
+      domain1Map.put("rcuSchemaPrefix", "jrfcdsmp");
+      // set the createDomainScriptsMountPath to non-default value
+      domain1Map.put("createDomainScriptsMountPath", "/u01/weblogic1");
+
+      // run RCU script to load db schema
+      DBUtils.runRCU(rcuPodName, domain1Map);
+
+      // create domain
+      logger.info("Creating Domain & verifying the domain creation");
+      domain1 = new JRFDomain(domain1Map);
+      domain1.verifyDomainCreated();
+
+    } finally {
+      if (domain1 != null && (JENKINS || testCompletedSuccessfully)) {
+        domain1.destroy();
+      }
+    }
+    logger.info("SUCCESS - " + testMethod);
+  }
+
+  /**
+   * This test case is to cover bug 29657663
```

Comment:
Please describe what the test is doing instead of referring to internal bugdb numbers